data-structures-and-algorithms-in-javascript/notes/09-javascript-tree-data-structure-in-javascript.md

```diff
@@ -0,0 +1,72 @@
+# Tree Data Structure in JavaScript
+
+**[📹 Video](https://egghead.io/lessons/javascript-tree-data-structure-in-javascript)**
+
+💻[Github repo](https://github.com/kyleshevlin/intro-to-data-structures-and-algorithms/blob/master/trees/index.js)
+
+## Summary
+
+![Tree data structure](../assets/Tree.png)
+
+A tree is a special type of graph - a collection of nodes and their relationships to other nodes. However, what makes a tree special is that it is hierarchical (in rank or order), and rather than neighbors, nodes have children.
```

Comment:
```suggestion
A tree is a special type of graph - a collection of nodes and their relationships to other nodes. **However, what makes a tree special is that it is hierarchical (in rank or order), and rather than neighbors, nodes have children.**
```